pennylane_qiskit/qiskit_device2.py

```diff
@@ -563,6 +532,7 @@ def generate_samples(self, circuit=None):
         Returns:
              array[complex]: array of samples in the shape ``(dev.shots, dev.num_wires)``
         """
+
```

Comment:
Is this added by `black`?

Reply:
Don't think so, probably added it by accident during merge conflicts